docs/crio.conf.5.md

```diff
@@ -215,10 +215,7 @@ the container runtime configuration.
 **ctr_stop_timeout**=30
   The minimal amount of time in seconds to wait before issuing a timeout regarding the proper termination of the container.
 
-**manage_network_ns_lifecycle**=false
-  **DEPRECATED** this option is being replaced by manage_ns_lifecycle, which is described below.
-
-**manage_ns_lifecycle**=false
+**manage_ns_lifecycle**=true
   Determines whether we pin and remove namespaces and manage their lifecycle
```

Comment:
I know it's pre-existing, but missing ending period.

Reply:
I was the one that wrote the original anyway :sunglasses: